src/test/java/internship/framework/core/utility/TestData.java

```diff
@@ -5,4 +5,11 @@
     private TestData() {
     }
 
+    //region yourDetails
+    public static final String YOUR_DETAILS_FIRST_NAME = "Radu";
```

Comment:
Why did we create a dedicated class just to store some constants? Are they related just for one test?

Reply:
TestData class was already part of Cristina's testing framework. She advised me to use that class for these constants.